website/content/docs/getting-started/update.md

```diff
@@ -7,41 +7,40 @@ weight: 2
 #### Update cluster
 
 1. Add a wordpress addon to the `blueprint.yaml`:
-```YAML
-- name: wordpress
-   kind: HelmAddon
-   enabled: true
-   namespace: wordpress
-   chart:
-      name: wordpress
-      repo: https://charts.bitnami.com/bitnami
-      version: 18.0.11
-```
+   ```YAML
+   - name: wordpress
+     kind: helm
+     enabled: true
+     namespace: wordpress
+     chart:
+       name: wordpress
+       repo: https://charts.bitnami.com/bitnami
+       version: 18.0.11
+   ```
 
 2. Update your cluster with the updated blueprint:
-
-```shell
-bctl update --config blueprint.yaml
-```
+   ```shell
+   bctl update --config blueprint.yaml
+   ```
 
 3. Verify that the wordpress addon is installed and running:
 
-```shell
-kubectl get pods --namespace wordpress
-```
+   ```shell
+   kubectl get pods --namespace wordpress
+   ```
 
-```shell
-NAME                           READY   STATUS      RESTARTS   AGE
-helm-install-wordpress-st8rh   0/1     Completed   0          2m58s
-wordpress-79d45fc94c-vg7n7     1/1     Running     0          2m49s
-wordpress-mariadb-0            1/1     Running     0          2m49s
-```
+   ```shell
+   NAME                           READY   STATUS      RESTARTS   AGE
+   helm-install-wordpress-st8rh   0/1     Completed   0          2m58s
+   wordpress-79d45fc94c-vg7n7     1/1     Running     0          2m49s
+   wordpress-mariadb-0            1/1     Running     0          2m49s
+   ```
 
 4. Connect to the wordpress page. You can forward requests to the server by running:
 
-```shell
-kubectl port-forward --namespace wordpress wordpress-79d45fc94c-vg7n7 8080:8080
-```
-> This command will need to be left running in the background. It does not return.
+   ```shell
+   kubectl port-forward --namespace wordpress wordpress-79d45fc94c-vg7n7 8080:8080
```

Comment:
I wonder if we should call out that this is the node name value

Reply:
> I wonder if we should call out that this is the node name value
Is it node name or pod name